src/app/personside/infotabs/utbetalinger/TotaltUtbetaltDetaljer.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { TotaltUtbetaltProps } from './TotaltUtbetalt';
+import { Undertekst } from 'nav-frontend-typografi';
+import { Utbetaling, Ytelse, Ytelseskomponent } from '../../../../models/utbetalinger';
+import { groupArray } from '../../../../utils/groupArray';
+import theme from '../../../../styles/personOversiktTheme';
+import {
+    filtrerBortUtbetalingerSomIkkeErUtbetalt,
+    formaterNOK,
+    getBruttoSumYtelser,
+    getNettoSumYtelser,
+    getPeriodeFromYtelser,
+    getTrekkSumYtelser
+} from './utbetalingerUtils';
+import { formaterDato } from '../../../../utils/dateUtils';
+import { Fragment } from 'react';
+import ErrorBoundary from '../../../../components/ErrorBoundary';
+
+const Wrapper = styled.div`
+  margin: 1.5rem 0 .5rem;
+  th {
+    font-weight: bold;
+  }
+  tr {
+    display: flex;
+    flex-flow: row wrap;
+    > *:first-child {
+      flex-grow: 1;
+    }
+    .sumNetto {
+      font-weight: bold;
+    }
+    .periodeForYtelse, .ytelseDetaljer {
+      flex-basis: 100%;
+      opacity: .7;
+    }
+    .periodeForYtelse {
+      text-align: left;
+    }
+    .ytelseDetaljer {
+      flex-grow: 1;
+      text-align: right;
+      margin-right: 12rem;
+      dl {
+        display: flex;
+        flex-flow: row wrap;
+        dt {
+          flex-basis: 50%;
+          flex-grow: 1;
+        }
+        dd {
+          padding-left: 1rem;
+          flex-basis: 7rem;
+        }
+      }
+    }
+  }
+  tbody tr > *:nth-last-child(n+3) {
+      padding-top: .5rem;
+      margin-top: .5rem;
+      border-top: 2px solid ${theme.color.bakgrunn};
+  }
+`;
+
+function getAlleYtelsesKomponenterFraYtelser(ytelser: Ytelse[]) {
+    return ytelser.reduce(
+        (acc: Ytelseskomponent[], ytelse: Ytelse) => {
```

Comment:
hva står acc for i denne sammenhengen?

Reply:
accumulated, trodde det var gnaske vanlig å bruke i sammenheng med reduce?

Reply:
ah, ja det er det sikkert :)